src/main/java/com/glodblock/github/proxy/ClientProxy.java

```diff
@@ -38,4 +60,34 @@ public void onLoadComplete(FMLLoadCompleteEvent event) {
             new DefaultExtractorLoader().run();
         }
     }
+
+    @SubscribeEvent
+    public void tickEvent(TickEvent.PlayerTickEvent e) {
+        super.tickEvent(e);
+        if (refreshTick + 1000 < System.currentTimeMillis()) {
```

Comment:
Make it configurable using `com.glodblock.github.common.Config.Config` and count ticks instead of milliseconds. You can just increment counter until it hit configured number and make it zero when restock cycle passed